index.html

```diff
@@ -76,15 +76,19 @@ <h1>Safari Trip Advisory</h1>
 	  <a class="next" onclick="plusSlides(1)">&#10095;</a>
 	</div>
 	<br>		
+	
+			<!--About_Us -->
 		
+<div id="About_us">

```

Comment:
A container for the About_us section was created to allow for green gradient background to be there...